addons/arsenal/stringtable.xml

```diff
@@ -433,5 +433,19 @@
             <Italian>Impossibile aprire l'arsenale ACE</Italian>
             <Japanese>ACE 武器庫を開けません</Japanese>
         </Key>
+          <Key ID="STR_ACE_Arsenal_Mission">
+            <English>ACE Arsenal</English>
+            <German>ACE-Arsenal</German>
+          </Key>
+          <Key ID="STR_ACE_Arsenal_ReturnToArsenal">
+            <English>Return to ACE Arsenal.</English>
+            <German>Zurück zum ACE-Arsenal.</German>
+          </Key>
+          <Key ID="STR_ACE_Arsenal_Mission_tooltip">
+            <English>Not as bad as the regular one.</English>
```

Comment:
Let's find something better than that.

Reply:
I'm all ears.

Reply:
`A more splendid Arsenal experience`

Reply:
No, we aren't using "splendid", we aren't BI!